### PR TITLE
Update Links in README

### PR DESCRIPTION
**Pull request by dricazenck**
Just minor adjustments in the README file. 

* Correct event link 
* Markdown cleanup 
* Add the slack channel link

## Type Of Change
- [x] Update documentation
- [ ] Add new event details
- [ ] Add project resources

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] Commit message is as per guidelines in the contributor guide

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -61,6 +65,8 @@ If you have resources you would like to share from your event or resources you t
     f) Update the `SLIDE_LINK` with url to slides (if speaker has shared) and `VIDEO_LINK` with the youtube url  
 
     g) Update the `SPEAKER_NAME` & `SPEAKER_TITLE`  
+    
+    h) Update the SLACK_CHANNEL_ID from Slack worspace
```
Comment (bharitwal) [inline CONTRIBUTING.md:69]: Small typo - 'workspace' spelling 
```suggestion
    h) Update the SLACK_CHANNEL_ID from Slack workspace
```

File: README.md [hunk 2]
```diff
@@ -84,9 +86,15 @@ We are looking for volunteers, and we prepared a detailed and comprehensive guid
...
-WWCode is the largest and most active community of technical women in the world. We have over 320,000 members in 147 countries & counting. Join our community by visiting [womenwhocode.com](womenwhocode.com) and sign up to become a member.
+WWCode is the largest and most active community of technical women in the world. We have over 320,000 members in 147 countries & counting. Join our community by visiting [womenwhocode.com](https://womenwhocode.com) and sign up to become a member.
  <br>
 
+## Mentorship Programme
+Mentorship is a fantastic way to get support for your personal growth and career development. 
+For mentors, it is a meaningful way of giving back and making a contribution. 
+
+See all details about the program [here](https://wwcode-mentorship.carrd.co).
+
```
Comment (bharitwal) [inline README.md:97]: This section should not be added to this readme:

- It should have its own directory under the resources folder with its own readme providing details about how to be involved. Similar to the readme under the Mock Interviews project. All projects supported by WWC-London would end up in the resources folder. Maybe I should rename 'resources' to  'projects'. Thoughts? 
- The link that is added here does not give details about the WWC-London mentorship program - dates, how to participate,  etc. These are just general guidelines for mentors and mentees.

**Review by bharitwal — CHANGES_REQUESTED** [review]
LGTM. Thanks for your contribution.

Added a few comments before I could merge the changes.

**Comment by dricazenck** [comment]
Sorry I will create another pr, because I cannot update this one anymore.

**Pull request closed without merging**